### PR TITLE
add property myproxy.enabled

### PR DESCRIPTION
**Pull request by ethaaalpha**
Adding the property inside the vip.conf to avoid the `Error reading proxy file /vip/.vip/proxies/biomed/x509up_server` when not using dirac.

### CODE REVIEW

File: vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java [hunk 1]
```diff
@@ -104,14 +104,17 @@ public String launch(String proxyFileName, String userDN)
             throws
             java.rmi.RemoteException,
             javax.xml.rpc.ServiceException {
+		String strProxy = null;
```
Comment (axlbonnet) [inline vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java:107]: seems you used tab indentation.
we use 4-spaces indent in vip.
Optionnaly you can try to convince me to change to tab but you need arguments.

Comment (axlbonnet) [reply vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java:107]: I think you initialize the var here as a good practice to declare all local variable at the beginning of the function ?

Comment (axlbonnet) [reply vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java:107]: Have you verified that `null` is supported and does not trigger an error in the vip/moteur communication ?

File: vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/proxy/ProxyClient.java [hunk 2]
```diff
@@ -95,7 +95,8 @@ public ProxyClient(Server server) {
     public void checkProxy() throws BusinessException {
 
         int proxyHoursValidity = server.getMyProxyMinHours();
-        if (proxyHoursValidity == 0) {
+
+        if (!server.getMyProxyEnabled()) {
```
Comment (axlbonnet) [inline vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/proxy/ProxyClient.java:99]: I like to add spaces around `!` to make it more visible if that's fine to you.

File: vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/SpringConfigServer.java [hunk 3]
```diff
@@ -244,6 +244,11 @@ public int getMyProxyMinHours() {
         return env.getRequiredProperty(CoreConstants.LAB_MYPROXY_MIN_HOURS, Integer.class);
     }
 
+	@Override
+	public boolean getMyProxyEnabled() {
+		return env.getRequiredProperty(CoreConstants.LAB_MYPROXY_ENABLED, Boolean.class);
```
Comment (axlbonnet) [inline vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/SpringConfigServer.java:249]: I had in mind to make it optional (but true by default).
And also to make the other `myproxy.*` config required only if `myproxy.enabled ` is true (or absent)

**Review by axlbonnet — CHANGES_REQUESTED** [review]
Very good work.
See the few details in comments :
- indentation style
- optional config if possible.

File: vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java (hunk 1 above)
Comment (ethaaalpha) [reply vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java:107]: > I think you initialize the var here as a good practice to declare all local variable at the beginning of the function ?

Yes

Comment (ethaaalpha) [reply vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/simulation/WebServiceEngine.java:107]: > Have you verified that `null` is supported and does not trigger an error in the vip/moteur communication ?

I've run few jobs on the vip portal and it all works well, the vip-errors.log and moteur_server.log were empty.

**Review by axlbonnet — APPROVED** [review]

**Pull request merged**